Makefile

```diff
@@ -47,6 +47,10 @@ ci: test-h2 test-sqlite test-postgres
 ephemeral: resources/public/admin
 	clojure -X:db-h2 lrsql.h2.main/run-test-h2 :persistent? false
 
+# like ephemeral, but takes env vars
+ephemeral-prod: resources/public/admin
+	clojure -M:db-h2 -m lrsql.h2.main --persistent false
```

Comment:
Have you tried using the regular `ephemeral` version and just add env-var-supporting vars to the `:test` aero profile?

Reply:
The idea here is that it is a quick way to launch the application _as it will be offered in the distribution_, so I don't see the utility of adding them statically like that. This lets me pass env vars from the cli without editing source.